public/index.php

```diff
@@ -67,4 +70,12 @@
     </div>
   </body>
 <?php endif; ?>
+<script>
+  $(function() {
+    app.user.id = "<?= isset($_SESSION['user']) ? $_SESSION['user']['id'] : '' ?>";
+    app.user.name = "<?= isset($_SESSION['user']) ? $_SESSION['user']['username'] : '' ?>";
+    app.user.email = "<?= isset($_SESSION['user']) ? $_SESSION['user']['email'] : '' ?>";
+    app.user.created_at = "<?= isset($_SESSION['user']) ? $_SESSION['user']['created_at'] : '' ?>";
+  })
+</script>
```

Comment:
make a correct merge with main, this parts are different in the main branch, yours commits are before the main branch so fix this